spark2/src/test/java/org/apache/iceberg/spark/source/TestSparkTableUtil.java

```diff
@@ -239,18 +238,13 @@ public void testImportWithNameMapping() throws Exception {
     List<String> actual = spark.read().format("iceberg").load(DB_NAME + ".target_table")
         .select("data")
         .sort("data")
-        .filter("data<'c'")
-        .collectAsList()
-        .stream()
-        .map(r -> r.getString(0))
-        .collect(Collectors.toList());
+        .filter("data >= 'b'")
```

Comment:
This would fail before the PR as the metrics were imported in the wrong way. The previous predicate did not trigger this issue, so I changed it.